backend/app/commands/DeleteResource.scala

```diff
@@ -32,7 +32,7 @@ class DeleteResource( manifest: Manifest, index: Index, previewStorage: ObjectSt
          // we introduced the OcrMyPdfExtractor.
          logger.info(s"No OCR languages found for blob ${uri.value}")
        } else {
-         logger.info(s"Deleting page previews for ${uri.value} in languages: ${ocrLanguages.map(_.key).mkString(", ")}")
+         // logger.info(s"Deleting page previews for ${uri.value} in languages: ${ocrLanguages.map(_.key).mkString(", ")}")
```

Comment:
remove?

Reply:
actually I think re-instate, they're still useful in prod